README.md

```diff
@@ -200,6 +200,30 @@ The command line flag takes precedence over any configuration you might have in
 
 > :point_right: **Bold keys** are mandatory or should be customized.
 
+### Connecting your IDE to QR
+
+Note that the Qontract Reconcile container will not start running until it is connected to by a debugger. You can use a config like this for VSCode:
+
+```json
+{
+  "name": "Python: Docker Attach",
+  "type": "debugpy",
+  "request": "attach",
+  "connect": {
+    "host": "127.0.0.1",
+    "port": 5678
+  },
+  "pathMappings": [
+    {
+      "localRoot": "${workspaceFolder}",
+      "remoteRoot": "."
+    }
+  ]
+}
+```
+
+[PyCharm instructions](https://www.jetbrains.com/help/pycharm/remote-debugging-with-product.html).
```

Comment:
this is not working for pycharm as we hard coded `debugpy` in qontract-reconcile, that support is tracked in https://youtrack.jetbrains.com/issue/PY-63403/Support-debugpy, this looks good but not released yet https://github.com/redhat-developer/lsp4ij/blob/main/docs/dap/user-defined-dap/python-debugpy.md

Reply:
so qontract-development-cli is not usable to debug from pycharm today ?

Reply:
no, I always switch to vscode when debug it

Reply:
Okay thanks for that context!